packages/zowe-explorer/CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 - Users can now update the credentials for a profile when the `autoStore` property is set to `false` in a team configuration. When `autoStore` is `false`, the new credentials are only saved in memory, and these credentials persist until the user updates them again or closes VS Code. [#3524](https://github.com/zowe/zowe-explorer-vscode/pull/3524)
 - Optimized the refresh functionality in Zowe Explorer for performance and stability. Now, the profiles are refreshed once before refreshing all the tree providers. [#3524](https://github.com/zowe/zowe-explorer-vscode/pull/3524)
 - Implemented the `SharedActions.refreshProvider` function to allow refreshing a single tree provider. [#3524](https://github.com/zowe/zowe-explorer-vscode/pull/3524)
+- Added script that will fetch latest localizations at every build now. [#3573](https://github.com/zowe/zowe-explorer-vscode/pull/3573)
```

Comment:
No need for a changelog since this may not be a user facing feature/bugFix

Reply:
Wouldn't the PR checks fail if I do not have it?

Reply:
The changelog check would fail, but we can mark as `no-changelog` and merge without bypassing any rules